src/controllerbluetooth.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+option=$1
+
+if [option = connect]
```

Comment:
Was this tested? In Bash, [ is a command so it needs spaces around it. You're also doing assignment instead of testing for equality. Just because Bash lets you do `option = connect`, doesn't mean you shouldn't specify that connect should be "connect"